infra/storage/valkey.tf

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "google_network_connectivity_service_connection_policy" "valkey_policy" {
+  provider      = google-beta
```

Comment:
I don't want to use the google-beta provider. We can keep this PR around until
1. google_network_connectivity_service_connection_policy and google_memorystore_instance move to the stable provider, or
2. We absolutely must move to valkey.

Reply:
This has now moved from BETA to GA. Check the release notes in this PR: https://github.com/GoogleChrome/webstatus.dev/pull/909